packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -101,11 +107,153 @@ export class UnfurlService {
         this.projectModel = projectModel;
     }
 
+    async getTitleAndDescription(
```

Comment:
not sure why these methods have moved through the file, so I can't see if there have been any changes.

Reply:
no changes but ESLint moved them on save 🤔